Dockerfile

```diff
@@ -1,8 +1,8 @@
-FROM python:3-alpine
+FROM python:3.7-slim
```

Comment:
Fixes ```No working compiler found, or bogus compiler options passed to
the compiler from Python's standard "distutils" module. See
the error messages above. Likely, the problem is not related
to CFFI but generic to the setup.py of any Python package that
tries to compile C code. (Hints: on OS/X 10.8, for errors about
-mno-fused-madd see http://stackoverflow.com/questions/22313407/
Otherwise, see https://wiki.python.org/moin/CompLangPython or
the IRC channel #python on irc.freenode.net.)```
during pip-install of requirements.

An alternative solution would be to find which `apk` packages must be installed for `python.3.7-alpine` to support installing CFFI during the resolving of the requirements dependencies.

Reply:
The switch to the `slim` is fine in my opinion.